packages/mdc-ripple/constants.js

```diff
@@ -15,42 +15,33 @@
  */
 
 export const ROOT = 'mdc-ripple';
-export const UPGRADED = `${ROOT}-upgraded`;
```

Comment:
nit: It seems like the exported const `ROOT` and `UPGRADED` is not being used anywhere.